Gemfile.jruby-1.9.lock

```diff
@@ -40,26 +30,41 @@ GEM
     ci_reporter_rspec (1.0.0)
       ci_reporter (~> 2.0)
       rspec (>= 2.14, < 4)
+    cinch (2.3.1)
     clamp (0.6.5)
-    coderay (1.1.0)
+    coderay (1.1.1)
     concurrent-ruby (0.9.2-java)
-    coveralls (0.8.10)
+    coveralls (0.8.13)
       json (~> 1.8)
-      rest-client (>= 1.6.8, < 2)
       simplecov (~> 0.11.0)
       term-ansicolor (~> 1.3)
       thor (~> 0.19.1)
       tins (~> 1.6.0)
     diff-lcs (1.2.5)
     docile (1.1.5)
-    domain_name (0.5.25)
+    domain_name (0.5.20160309)
```

Comment:
I have checked why, its for a internal reference when they update TLD database.. but oh well